__tests__/app.test.js

```diff
@@ -22,4 +22,23 @@ describe.only('GET /api/categories', () => {
             })
         })
     })
+})
+
+describe('GET /api', () => {
+    test('/api - status -200 - respond with an object containing all the endpoints a user can make', () => {
+        return request(app).get('/api').expect(200).then(response => {
+            expect(typeof response.body.endpointData).toBe('object')
+        })
+    })
+    test('/api - status -200 - respond with an object that has an endpoint key, the value of that key being a description key, query key and an example response key', () => {
+        return request(app).get('/api').expect(200).then(response => {
+        
+        Object.values(response.body.endpointData).forEach(endpoint => {
+            expect(endpoint.hasOwnProperty('description')).toBe(true)
+            expect(endpoint.hasOwnProperty('queries')).toBe(true)
```

Comment:
This is fine, however, you could instead require in the same JSON file and assert it is sent as a response, in order to test more accurately :)